INFO:     Started server process [1407228]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://localhost:9000 (Press CTRL+C to quit)
start FastAPI server at localhost:9000
INFO:     127.0.0.1:50370 - "POST /process_batch HTTP/1.1" 200 OK
INFO:     127.0.0.1:39070 - "POST /process_batch HTTP/1.1" 200 OK
INFO:     127.0.0.1:55188 - "POST /process_batch HTTP/1.1" 200 OK
INFO:     Shutting down
INFO:     Waiting for background tasks to complete. (CTRL+C to force quit)
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [1407228]
调用engine时出错: HTTPConnectionPool(host='localhost', port=9001): Max retries exceeded with url: /generate_text_batch (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x14d08ddc9cf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
调用engine时出错: HTTPConnectionPool(host='localhost', port=9001): Max retries exceeded with url: /generate_text_batch (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x14d08ddca350>: Failed to establish a new connection: [Errno 111] Connection refused'))
